test/workers/create_service_token_worker_test.rb

```diff
@@ -51,7 +51,7 @@ def setup
     attr_reader :service, :user, :event
 
     def test_perform_deserialization_error_no_service
-      expected_log_message = /CreateServiceTokenWorker#perform raised ActiveJob::DeserializationError with message: Error while trying to deserialize arguments: Couldn't find Service with 'id'=#{service.id}/
+      expected_log_message = /CreateServiceTokenWorker#perform raised ActiveJob::DeserializationError with message: Error while trying to deserialize arguments: Couldn't find Service with 'id'="#{service.id}"/
```

Comment:
Apparently, one of the CVEs fixed modifies the way data is logged:

https://discuss.rubyonrails.org/t/cve-2025-55193-ansi-escape-injection-in-active-record-logging/89669

This change to add quotes is now required, Rails developer did the same with their tests:

https://github.com/rails/rails/commit/3beef20013736fd52c5dcfdf061f7999ba318290#diff-1721af6d2f906e7e5eec9cac9a7707cd051c1a1f5ffb1b38c69a6c5a52ae4e9aR1701